tools/spec-gen-sdk/src/types/sdks.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * This file is the single source of truth for the labels used by the SDK generation tooling
+ * in the Azure/azure-rest-api-specs and Azure/azure-rest-api-specs-pr repositories.
+ *
+ * For additional context, see:
+ * - https://gist.github.com/raych1/353949d19371b69fb82a10dd70032a51
+ * - https://github.com/Azure/azure-sdk-tools/issues/6327
+ * - https://microsoftapc-my.sharepoint.com/:w:/g/personal/raychen_microsoft_com/EbOAA9SkhQhGlgxtf7mc0kUB-25bFue0EFbXKXS3TFLTQA
+ */
+export type SdkName =
+  | "azure-sdk-for-go"
+  | "azure-sdk-for-java"
+  | "azure-sdk-for-js"
+  | "azure-sdk-for-net"
+  | "azure-sdk-for-net-track2"
```

Comment:
could you remind me the difference between `azure-sdk-for-net` and `azure-sdk-for-net-track2` ?

Reply:
I would love to remove this "-track2" as part of this work if possible. IIRC the older was for the old code generator.

Reply:
same. This causes so much confusion that if we can use this effort to clean it up, I am all up for it!

Reply:
Thank you for the suggestion. I like it. I will confirm with Arthur if the name can be updated. Otherwise, I prefer to keep the name consistent with what they use.

Reply:
I have initiated the discussion with Shanghai team on this name repurpose and will update this code if decided.

Reply:
Thank you!